app/facades/trust_registry.py

```diff
@@ -1,25 +1,120 @@
 import logging
 import os
-from fastapi.exceptions import HTTPException
-import requests
-from typing import List, Literal
+from typing import List, Literal, Optional, TypedDict
 
+import requests
+from fastapi.exceptions import HTTPException
 
-TRUST_REGISTRY_URL = os.getenv("TRUST_REGISTRY_URL", "http://localhost:8001/")
+TRUST_REGISTRY_URL = os.getenv("TRUST_REGISTRY_URL", "http://localhost:8001")
 
 logger = logging.getLogger(__name__)
 
 Role = Literal["issuer", "verifier"]
 
 
+class TrustRegistryException(Exception):
+    """Class that represents a trust registry error"""
+
```

Comment:
If you don't have any extra logic here that you want, maybe just adding `pass` makes it clearer.